debian/snapd.postrm

```diff
@@ -53,6 +53,13 @@ if [ "$1" = "purge" ]; then
         rm -f "/etc/systemd/system/multi-user.target.wants/$unit"
     done
 
+    echo "Discarding preserved snap namespaces"
+    # opportunistic as those might not be actually mounted
+    for mnt in /run/snapd/ns/*.mnt; do
+        umount --lazy "$mnt" || true
+    done
+    umount --lazy /run/snapd/ns/ || true
```

Comment:
Done, thanks, I missed this earlier. I changed all occurrences of `umount --lazy` to `umount -l`